devdox/app/routes/api_keys.py

```
@@ -49,3 +53,22 @@ async def revoke_api_key(
 
     return APIResponse.success(message=constants.API_KEY_REVOKED_SUCCESSFULLY)
+
+
+@router.get(
+    "/",
+    response_model=Dict[str, Any],
+    status_code=status.HTTP_200_OK,
+    summary="Retrieve user api keys",
+    description="Retrieves the API Key per user",
+)
+async def revoke_api_key(
```

Flake8 (7.2.0)

[error] redefinition of unused 'revoke_api_key' from line 46

(F811)